db/migrate/20230221125618_create_analytics_events.rb

```diff
@@ -0,0 +1,12 @@
+class CreateAnalyticsEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :analytics_events do |t|
+      t.string :event_type
+      t.string :page
+      t.string :assessment_code
+      t.string :browser_id
```

Comment:
do we want some null: falses and some indexes in here?

Reply:
Cool. I think the final say about the right indexes will come out of EL-717, but as a start I think it makes sense to index the created_at and assessment_code columns, as a lot of queries are based around those.